person.rb

```diff
@@ -3,14 +3,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unkown', parent_permission = true) # rubocop:disable Style/OptionalBooleanParameter
     @id = Random.rand(1..1000)
     super()
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def can_use_services?
```

Comment:
Good job so far, but you're missing out on:

- [ ] A method for adding rentals. It's required that you have a method for adding rental in this class. Kindly work on that 👍